clean_python/fastapi/request_query.py

```diff
@@ -44,7 +44,10 @@ def filters(self) -> List[Filter]:
         for name in self.model_fields:
             if name in {"limit", "offset", "order_by"}:
                 continue
+            # deal with list query paramerers
             value = getattr(self, name)
             if value is not None:
-                result.append(Filter(field=name, values=[value]))
+                if not isinstance(value, List):
```

Comment:
```suggestion
                if not isinstance(value, list):
```

En zijn lists de enige optie? Kunnen er geen tuples / sets/ andere sequences in komen?

+ dit heeft een test nodig